src/main/java/frc/robot/Robot.java

```diff
@@ -89,6 +89,7 @@ public void robotPeriodic() {
   @Override
   public void disabledInit() {
     CommandScheduler.getInstance().cancelAll();
+    // TODO: This method was called in robotInit(); is it needed here?
```

Comment:
yes it is needed there to create new pathplanner commands as they only run once per object